instrumentation/gruf/Appraisals

```diff
@@ -4,10 +4,7 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-appraise 'gruf-2.15.1' do
```

Comment:
This means we lose test coverage for versions prior to 2.19.

Is that ok? Are you looking to drop support for older versions?

Reply:
I added the support of the last three versions of Gruf, it seems that there is no need to indicate the dependence of the GRPC version

Reply:
thank you!